en/02_Developer_Guides/15_Customising_the_Admin_Interface/06_Javascript_Development.md

```diff
@@ -343,32 +219,6 @@ $('form').bind('submit', function(e) {
 });
 ```
 
-See [interactive example on jsbin.com](http://jsbin.com/opuva)
-
-You can also use the [jQuery.metadata Plugin](http://docs.jquery.com/Plugins/Metadata/metadata) to serialize data into
-properties of DOM elements. This is useful if you want to encode element-specific data in markup, for example when
```

Comment:
Our JSbin examples don't seem to be there anymore.

jQuery.metadata doesn't seem to be a thing anymore.